src/less/filter-menu/base/filter-menu.less

```diff
@@ -0,0 +1,99 @@
+span.filter-menu {
+    display: inline-block;
+    max-width: 288px;
+    min-width: 144px;
+}
+
+.filter-menu__items {
+    .dropdown-base;
+
+    border-color: @filter-menu-border-color;
+    border-radius: 16px;
+    overflow: hidden;
+}
+
+.filter-menu__scroll-area {
+    margin-top: 8px;
+    max-height: calc(50vh - 40px); // half of the viewport height minus the footer
+    overflow-y: auto;
+    position: relative;
+}
+
+span.filter-menu__items[role="menu"] {
```

Comment:
Should we keep the roles? I remember ian removed them.

Reply:
This is a pretty close copy of the menu, which Ian already had redone.

Reply:
Remember, we need two versions of the filter-menu. One with ARIA semantics, one without (i.e. a form).

Reply:
I thought we were going to do two different roots for each. Like filter-menu-form (we came up with something else i believe)  That way we don't have to worry about which selectors are more specific. 
Im fine keeping this approach but i think it can get a bit verbose and confusing which styles override others.

Reply:
I thought we were going to do two different roots for each. Like filter-menu-form (we came up with something else i believe)  That way we don't have to worry about which selectors are more specific. 
Im fine keeping this approach but i think it can get a bit verbose and confusing which styles override others.

Reply:
Yep, two different roots for each to make the usage clear.